src/OpenTelemetry.Exporter.OpenTelemetryProtocol/LogRecordExportProcessorOptions.cs

```diff
@@ -0,0 +1,60 @@
+// <copyright file="LogRecordExportProcessorOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Microsoft.Extensions.Configuration;
+using OpenTelemetry.Internal;
+using OpenTelemetry.Logs;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol;
+
+/// <summary>
+/// Options for configuring either a <see cref="SimpleLogRecordExportProcessor"/> or <see cref="BatchLogRecordExportProcessor"/>.
+/// </summary>
+public class LogRecordExportProcessorOptions
+{
+    private BatchExportLogRecordProcessorOptions batchExportProcessorOptions;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="LogRecordExportProcessorOptions"/> class.
+    /// </summary>
+    public LogRecordExportProcessorOptions()
+        : this(new ConfigurationBuilder().AddEnvironmentVariables().Build())
```

Comment:
This could be simplified. `BatchExportLogRecordProcessorOptions` class calls `new ConfigurationBuilder().AddEnvironmentVariables().Build()` anyway.

https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Logs/BatchExportLogRecordProcessorOptions.cs#L44

Reply:
If I understand it correctly, it is needed to do https://github.com/open-telemetry/opentelemetry-dotnet/blob/039556d1833c5d3d0e122e5090c27e366919e0f4/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpTraceExporterHelperExtensions.cs#L73

@CodeBlanch Please correct if my understanding is wrong

Reply:
change pushed by @CodeBlanch